src/config.ts

```diff
@@ -210,7 +210,7 @@ function validateAlertThreshold(alertThreshold: number | null, failThreshold: nu
     if (alertThreshold === null) {
         throw new Error("'alert-threshold' input must not be empty");
     }
-    if (failThreshold && alertThreshold > failThreshold) {
+    if (failThreshold && Math.abs(alertThreshold) > Math.abs(failThreshold)) {
```

Comment:
Is this a breaking change as it adds another scenario that can alert?

The breaking change may be prevented if we add an additional configrable field to toggle this.

WDYT? How strict are we treating breaking change? @ktrz

Reply:
I'm not super fond of the current solution. I would much rather modify the `biggerIsBetter` function to determine the value based on the units as well.